src/Workspaces/Remote/Core/ProjectSystem/IWorkspaceProject.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using StreamJsonRpc;
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem
+{
+    [RpcMarshalable]
+    public interface IWorkspaceProject : IAsyncDisposable
+    {
+        public Task SetDisplayNameAsync(string displayName, CancellationToken cancellationToken);
```

Comment:
Seems like the access modifiers here aren't needed?

Reply:
Nope, that was brain on autopilot mode.